README.md

```diff
@@ -64,13 +64,15 @@ configuration file.
-| **_github_dimmed_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/alacritty/github_dimmed.yml)                          | ![github_dimmed](images/github_dimmed.png)                     |
-| **_github_light_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/alacritty/github_light.yml)                            | ![github_light](images/github_light.png)                       |
-| **_github_light_colorblind_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/alacritty/github_light_colorblind.yml)      | ![github_light_colorblind](images/github_light_colorblind.png) |
-| **_github_light_default_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/alacritty/github_light_default.yml)            | ![github_light_default](images/github_light_default.png)       |
+| **_github_dark_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_dark.yml)                          | ![github_dark](images/github_dark.png)                         |
+| **_github_dark_colorblind_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_dark_colorblind.yml)  | ![github_dark_colorblind](images/github_dark_colorblind.png)   |
+| **_github_dark_dimmed_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_dark_dimmed.yml)          | ![github_dark_dimmed](images/github_dark_dimmed.png)           |
+| **_github_dark_high_contrast_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_dark_high_contrast.yml)| ![github_dark_high_contrast](images/github_dark_high_contrast.png)|
+| **_github_dark_tritanopia_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_dark_tritanopia.yml)  | ![github_dark_tritanopia](images/github_dark_tritanopia.png)   |
+| **_github_light_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_light.yml)                      | ![github_light](images/github_light.png)                       |
+| **_github_light_colorblind_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_light_colorblind.yml)| ![github_light_colorblind](images/github_light_colorblind.png) |
+| **_github_light_high_contrast_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_light_high_contrast.yml)| ![github_light_high_contrast](images/github_light_high_contrast.png)|
+| **_github_light_tritanopia_**<br>[source](https://github.com/projekt0n/github-theme-contrib/blob/main/themes/alacritty/github_light_tritanopia.yml)| ![github_light_tritanopia](images/github_light_tritanopia.png) |
```

Comment:
You've added more themes here, but it doesn't look like you've added any themes?

Reply:
My bad, I forgot to stage the other files. One thing to note: I've renamed github_dimmed to github_dark_dimmed so that it's consistent with the official name of the theme and the source file for the theme. This would require intervention from any users who update the repo while using github_dimmed, so if it needs to be reverted let me know.

Reply:
Thanks for the heads-up. We're not far away from the toml update anyway which will break all configs, so I think it should be fine.